samples/bluetooth/peripheral_hids_keyboard/src/main.c

```diff
@@ -856,6 +856,8 @@ static void button_changed(uint32_t button_state, uint32_t has_changed)
 
 	uint32_t buttons = button_state & has_changed;
 
+	printk("Button state changed: 0x%08x \n", buttons);
```

Comment:
We already have a log for every press action of the "advertising start" button. With each button press of this particular button you will get one of the following messages:

- "Advertising successfully started"
- "Advertising failed to start (err %d)"
- "Advertising continued"